src/sentry/ownership/grammar.py

```diff
@@ -379,7 +379,7 @@ def parse_code_owners(data: str) -> Tuple[List[str], List[str], List[str]]:
         if rule.startswith("#") or not len(rule):
             continue
 
-        assignees = rule.strip().split()[1:]
+        assignees = [i for i in re.split(r"(?<!\\)\s", rule.strip()) if i][1:]
```

Comment:
@NisanthanNanthakumar Could you make the regex a constant or variable with it's own name to identify what it's meant to be matching? I know from the PR title it's spaces, but that's not obvious in the code.